OpenSim/Simulation/Model/Model.cpp

```diff
@@ -661,9 +661,15 @@ void Model::createMultibodyTree()
 
     auto bodies = getComponentList<Body>();
     for (auto& body : bodies) {
-        _multibodyTree.addBody( body.getAbsolutePathName(),
-                                body.getMass(), false,
-                                const_cast<Body*>(&body) );
+        try {
+            _multibodyTree.addBody( body.getAbsolutePathName(),
+                                    body.getMass(), false,
+                                    const_cast<Body*>(&body) );
+        } catch (const std::runtime_error& e) {
+            cout << "Duplicate Body names are not permitted. Please update "
+                 << "your model's .osim file accordingly." << std::endl;
```

Comment:
> In other places (e.g., here), we throw a new exception like so:

I am familiar with that paradigm, but I believe it is a different situation because the original exception was thrown by OpenSim's `findAndConnect()`. In model.cppL668, the original exception was thrown by Simbody. The exception is a generic `runtime_error` and OpenSim cannot be certain (now and in the future) that the exception was thrown because a duplicate body name (absolute path name) was found.

There are several possible designs here. One option is to assume the `runtime_error` was thrown because a duplicate name was found and re-throw an OpenSim-specific exception. A second option is to search for the word "duplicate" in the string associated with Simbody's exception—proceed as in the first option if found; otherwise, add a message like "perhaps there are duplicate absolute path names" and then throw the original exception. Please let me know which you prefer, or if you prefer a third option.